oryx-app-mllib/src/main/java/com/cloudera/oryx/app/mllib/kmeans/KMeansUpdate.java

```diff
@@ -137,7 +134,6 @@ public double evaluate(JavaSparkContext sparkContext,
                          PMML model,
                          Path modelParentPath,
                          JavaRDD<String> testData) {
-    // TODO: A Strategy pattern to be able to invoke different evaluation metrics
     JavaRDD<Vector> testingData = parsedToVectorRDD(testData.map(MLFunctions.PARSE_FN));
     return pmmlToKMeansModel(model).computeCost(testingData.rdd());
```

Comment:
I think this is the wrong way around since it returns a cost, which is higher when the clustering is worse. I can address that with a few other additional tests in a follow-on change.